README.md

```diff
@@ -20,11 +20,11 @@ https://github.com/DataBiosphere/data-explorer-indexers, or any other indexer.
 * Create `api/dataset_config/<my dataset>`
   * If you used https://github.com/DataBiosphere/data-explorer-indexers, copy
     the config directory from there.
-  * If you used your own indexer, the config files must follow
-    this format ([see examples](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/dataset_config/platinum_genomes)). Specifically:
-    * `dataset.json` has a `name` field.
-    * `facet_fields.csv` has the `project_id, dataset_id, table_name, readable_field_name`
-    columns filled out.
+  * Copy and fill out [ui.json](https://github.com/DataBiosphere/data-explorer/tree/master/api/dataset_config/template/ui.json).
```

Comment:
Could we document somewhere that ui.json contains the config for specifying which facets you want to see, and you should fill it out with whatever you're interested in looking at? I don't think that's explicitly mentioned anywhere but isn't obvious to me walking through the steps and filling out config.

Reply:
I added a line to `ui.json`. By convention, comments go in the .json files. This way, they'll always see the comment; as opposed to only seeing the comment if they read the README first.